pages.de/common/docker.md

```diff
@@ -3,6 +3,10 @@
 > Zum Verwalten von Docker Containern und Images.
 > Mehr Informationen: <https://docs.docker.com/engine/reference/commandline/cli/>.
 
+- Baut einen Container über eine Dockerfile im selben Verzeichnis
```

Comment:
The translation should be made from the primary (English) pages.

For docker we have specific subcommand pages like `docker-build.md`, `docker-start.md` etc.

If you want to add or edit examples you should send a PR for edit the primary page, and you can include the update for the German translation for the same page in that PR.